src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -97,7 +97,7 @@ public static function createSystemCache(string $namespace, int $defaultLifetime
             return $opcache;
         }
 
-        if (\in_array(\PHP_SAPI, ['cli', 'phpdbg'], true) && !filter_var(\ini_get('apc.enable_cli'), \FILTER_VALIDATE_BOOL)) {
+        if ('cli' === \PHP_SAPI && !filter_var(\ini_get('apc.enable_cli'), \FILTER_VALIDATE_BOOL)) {
```

Comment:
aligning with other checks we have in place for APCu (not need to enable it for phpdbg/embed)